### PR TITLE
Changing to axios as http calls manager

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
cc/ @sdesimone so that he can try or object as it may affect other products based on these bindings.

**Comment by mmerce** [comment]
retest this, please

File: lib/BaseScript.js [hunk 1]
```diff
@@ -16,12 +16,14 @@
...
-var Resource = require('./Resource');
-var request = require('request');
-var constants = require('./constants');
-var logger = require('./logger');
-var utils = require('./utils');
+const axios = require('axios');
+const https = require('https');
+const BigML = require('./BigML');
+const Resource = require('./Resource');
+const request = require('request');
```
Comment (sdesimone) [inline lib/BaseScript.js:23]: is this `require` necessary?

Comment (mmerce) [reply lib/BaseScript.js:23]: No, good catch!

**Review by jaor — APPROVED** [review]

**Pull request merged**